### PR TITLE
docs: adds docs on generic runtime variables

### PR DESCRIPTION
**Pull request by tnolet**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer
This was not in our docs yet but was live in the editor and backend.

### CODE REVIEW

File: site/content/docs/api-checks/setup-teardown-scripts.md [hunk 1]
```diff
@@ -154,6 +154,18 @@ Response properties are exposed a standard Javascript object. These are only ava
 | `response.timings`  | Various timestamps for the request stage relative to the starting time.  | Object |
 | `response.timingPhases`  | Time durations for each request phase. | Object |
 
+### General runtime variables
+
+The setup and teardown runtime also exposes a set of generic variables you can use to figure out what
```
Comment (stefanjudis) [inline site/content/docs/api-checks/setup-teardown-scripts.md:159]: May I propose to make it shorter? 😊

> The setup and teardown runtime also exposes a set of environment variables (e.g. `process.env.CHECK_NAME`) to figure out what check, check type etc. you are running.

File: site/content/docs/api-checks/setup-teardown-scripts.md [hunk 2]
```diff
@@ -154,6 +154,18 @@ Response properties are exposed a standard Javascript object. These are only ava
...
+### General runtime variables
+
+The setup and teardown runtime also exposes a set of generic variables you can use to figure out what 
+check, check type etc. you are running. Again, these are accessible by using `process.env.CHECK_NAME` for example.
+
+| property                  | description                                                | type   |
+|---------------------------|------------------------------------------------------------|--------|
+| `CHECK_NAME`              | The name of the check being executed.                      | String |
+| `CHECK_ID`                | The UUID of the check being executed.                      | String |
+| `CHECK_TYPE`              | The type of the check being executed.                      | String |
```
Comment (stefanjudis) [inline site/content/docs/api-checks/setup-teardown-scripts.md:166]: When I saw this I was like "hmm, is this then `API` or `Browser`?"). Putting the options in `()` might be helpful. :)

File: site/content/docs/browser-checks/variables.md [hunk 3]
```diff
@@ -91,3 +91,14 @@ You can make use of this by providing a default value for a specific variable at
 be overridden at the check level.
 
 
+## General built-in runtime variables
+
+The Browser Check runtime also exposes a set of generic variables you can use to figure out what
```
Comment (stefanjudis) [inline site/content/docs/browser-checks/variables.md:96]: Same. :) 

> The Browser Check runtime also exposes a set of environment variables (e.g. process.env.CHECK_NAME) to figure out what check, check type etc. you are running.

File: site/content/docs/api-checks/setup-teardown-scripts.md (hunk 1 above)
Comment (tnolet) [reply site/content/docs/api-checks/setup-teardown-scripts.md:159]: @stefanjudis sure go for it.

Comment (stefanjudis) [reply site/content/docs/api-checks/setup-teardown-scripts.md:159]: Haha. My proposal was actually in the comment. :) I'll push and approve. :)

**Review by stefanjudis — APPROVED** [review]
:shipit: 💯

**Pull request merged**